Resources/doc/features/user-context.rst

```diff
@@ -36,7 +36,17 @@ These five steps resemble the Overview in the FOSHttpCache documentation.
 5. If the response to ``/foo`` should differ per user context, the application
    sets a ``Vary: X-User-Context-Hash`` header. The appropriate user context
    dependent representation of ``/foo`` will then be returned to the client.
+   
+.. note::
 
+    This concept basically executes a "preflight request" ("preflight" was term
```

Comment:
fair enough. CORS is a valid reference for me to tell whether a word is an actual word :-)
please add preflight to the additional words list: https://github.com/Toflar/FOSHttpCacheBundle/blob/patch-1/Resources/doc/spelling_word_list.txt . that should make the build green

can you also check the grammar of what you say in paranthesis? "was term was...". maybe "(a preflight request is a request that is sent prior to the real request - this is also done for `CORS requests`_ for example)"